src/docfx.website.themes/docs.html/ManagedReference.mta.json.js

```diff
@@ -62,7 +62,8 @@ exports.transform = function (model) {
     "isEii",
     "isExtensionMethod",
     "nameWithType",
-    "extensionMethods"
+    "extensionMethods",
+    "baseRepositoryDirectory"
   ];
```

Comment:
we may need to filter in Conceptual as well as in op.html, since the metadata is set globally regardless of document type